astro/src/content/docs/customize/email-and-messages/message-templates.mdx

```diff
@@ -53,9 +53,48 @@ Most templates will contain the User object as returned on the Retrieve User API
 
 Below you will find each stock template that FusionAuth ships for reference. The available replacement values will be outlined below for each template.
 
+## Default Forgot Password
```

Comment:
I've been adding these to `templates-replacement-variables.mdx` as well. Should this be done there too? I've also been cleaning up the existing components (see my last few site commits) that use the `RemoteCode` component as well.

Reply:
That page is for email template variables. These are for message template variables. There are no changes to email templates for this.

Reply:
oops. got my wires crossed. I might have started a bad trend then. On the passowrdless form docs change (e77e6a96064f1f0056a12e5ac42410f46631b613), there is a legitimate reason on `astro/src/content/docs/lifecycle/authenticate-users/passwordless/magic-links.mdx` to show the message and phone templates at the same time.

I think I continued that pattern too far into the templates-replacement-variables page. Captured https://linear.app/fusionauth/issue/ENG-2670/remove-the-message-templates-from-templates-replacement-variablesmdx to go back and fix that.